home/import_assessments.py

```diff
@@ -457,6 +459,18 @@ def check_score_type(
             )
 
 
+def check_score_field(score: Any, row_num: int) -> None:
+    if score:
+        try:
+            [float(s) for s in score.split(",")]
```

Comment:
We use `deserialise_list` for this on the actual check, which might interpret this differently than `split`, so maybe we should use the same method here?